receiver/prometheusremotewritereceiver/receiver_test.go

```diff
@@ -471,6 +471,73 @@ func TestTranslateV2(t *testing.T) {
 				return metrics
 			}(),
 		},
+		{
```

Comment:
Please add a testcase for classic histogram as well to cover both cases. Seems to be missing currently.

Reply:
My understanding is that classic histograms are impossible to support due to the sharding strategy of Prometheus Remote-Write.

Once a batch of metrics in Prometheus gets large enough, it is split into several remote-write requests. Since classic histograms are not atomic, i.e., they are a bunch of time series that together represent a histogram, there's a big chance that different buckets come in separate requests.

It's impossible to tell if we received all the buckets of a histogram because some might get lost on the wire, so even caching isn't an option here.

Reply:
I will rename this PR to be: `add exponential histograms datapoints`. Then we can go forward. If we need to take care of native and classic histograms here. Then I can open a new PR.

Reply:
> My understanding is that classic histograms are impossible to support due to the sharding strategy of Prometheus Remote-Write.

Ah, I see. Makes sense why I didn't see tests covering classic histograms.
But then what did `addHistogramDatapoints` actually do here?

Sidenote: there will be a way to support classic histograms in the future in some sense with RW2. @bwplotka, me and others came up with native histograms with custom buckets, Bartek's use case was pretty much this.  See https://github.com/prometheus/proposals/blob/main/proposals/2024-01-26_classic-histograms-stored-as-native-histograms.md